commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/ClayTableAction.java

```diff
@@ -23,13 +23,30 @@ public ClayTableAction(
 		String href, String icon, String label, boolean quickAction,
 		boolean separator) {
 
+		_cssClasses = "";
```

Comment:
Should be `StringPool.BLANK` rather than `""`.

Reply:
Also, it'd be better to have this constructor simply call the new constructor, with the default value passed in -- that way it's obvious there's only that simple difference between the constructors (these ones are pretty simple so it's not much different at the moment, but if they get any more complicated in the future the difference would be significant).